src/EPR.CommonDataService.Api/Controllers/ProducerPropertiesController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using EPR.CommonDataService.Api.Configuration;
```

Comment:
Please rename the file to Producer**Details**Controller.cs to match the class name.

Reply:
done